migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,84 @@
+[id="migration-from-version-3-to-4-overview"]
+= Migration from version 3 to 4 overview
+
+{product-title} 4 clusters are different from {product-title} 3 clusters. {product-title} 4 clusters contain new technologies and functionality that result in a cluster that is self-managing, flexible, and automated To know more about migrating from {product-title} 3 to 4 see, xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[migrating from version 3 to 4].
+
+[discrete]
+[id="mtc-3-to-4-overview-differences-mtc"]
+=== Differences between {product-title} 3 and 4
+Differences between {product-title} 3 and 4: To successfully transition to {product-title} 4, review the following information:
+
+** Architecture
+** Installation and upgrade
+** Storage, network, logging, security, and monitoring considerations
+
+[discrete]
+[id="mtc-3-to-4-overview-planning-network-considerations-mtc"]
+=== Planning network considerations
+Planning network considerations: Before migrating from {product-title} 3 to 4 plan your network considerations, review the following information:
+
+** DNS considerations
+*** Isolating the DNS domain of the target cluster from the clients.
+*** Setting up the target cluster to accept the source DNS domain.
+
+You can migrate stateful application workloads from {product-title} 3 to 4 at the granularity of a namespace. To know more about MTC see the xref:../migration_toolkit_for_containers/about-mtc.adoc#about-mtc[Understanding MTC].
+
+[NOTE]
+====
+If you are migrating from {product-title} 3, see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from {product-title} 3 to 4] and xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Installing the legacy {mtc-full} Operator on {product-title} 3].
+====
+
+[discrete]
+[id="mtc-overview-install-mtc"]
+== Installating MTC
+You must install the Migration Toolkit for Containers that is compatible with your {product-title} version by using OLM. You manually install the legacy Migration Toolkit for Containers Operator on {product-title} 4.5 and earlier versions. You must configure an object storage to use as a replication repository. You can check the following items:
```

Comment:
The first sentence is confusing because it suggests that each OCP version has its own MTC version and that OLM installs the Operator on all versions. Both are incorrect. There are only two MTC Operator versions, 1.6 (latest version, on OLM) and 1.5 (aka Legacy Operator, pulled from registry.redhat.com).

Suggest you change this to: "Install the Migration Toolkit for Containers Operator on your target cluster by using OLM. Install the Legacy Migration Toolkit for Containers Operator manually on your source cluster. Then configure object storage to use as a replication repository. 

You can format this as a procedure if you think it looks better.

Do not mention 4.5 and earlier because the user is migrating from 3 to 4. So we assume that the source cluster is 3.11 and that the target cluster is the current OCP version.

Reply:
Addressed